### PR TITLE
add verenigingen criterion

### PR DESCRIPTION
**Pull request by wolfderechter**
## ID
DGS-292

## Description
Add new verenigingen criterion

## Type of change

 - [ ] Bug fix
 - [x] New feature
 - [ ] Breaking change
 - [ ] Maintanance

## How to setup
...

## How to test
...

### CODE REVIEW

**Review by cecemel — CHANGES_REQUESTED** [review]
besides the remarks, it should work. but the proof of the pudding will be in the eating.

**Review by cecemel — APPROVED** [review]
thanks. We'll soon find out if it works

**Review by cecemel — CHANGES_REQUESTED** [review]

File: config/migrations/2024/20240715104628-add-verenigingen-criterion.sparql [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  GRAPH <http://mu.semte.ch/graphs/public> {
+    # Create ACMIDMOrganisationClassification concept scheme
+    <http://data.lblod.info/id/conceptscheme/ACMIDMOrganisationClassification> a <http://www.w3.org/2004/02/skos/core#ConceptScheme>;
+      mu:uuid "586e1a3d-56ef-424b-862d-2123d8b7189f" ;
+      skos:prefLabel "ACM/IDM OrganisatieClassificatie"@nl ;
+      skos:prefLabel "ACM/IDM OrganisationClassification"@en ;
+      skos:definition "Lijst van type organisaties binnen ACM/IDM"@nl .
+
+    # Add concept-scheme for economische actoren
+    <http://data.lblod.info/id/conceptscheme/c6157470-9fb3-4e8d-a9b7-8737dbfa3642>
```
[the remark this answers is not in the document]
Comment (wolfderechter) [reply config/migrations/2024/20240715104628-add-verenigingen-criterion.sparql:18]: For `<http://data.lblod.info/id/conceptscheme/ACMIDMOrganisationClassification>` as well?

[the remark this answers is not in the document]
Comment (cecemel) [reply config/migrations/2024/20240715104628-add-verenigingen-criterion.sparql:18]: no that last one makes sense. You have skos:Concept and thats != to a skos:Conceptscheme

**Review by cecemel — CHANGES_REQUESTED** [review]

File: config/migrations/2024/20240715104628-add-verenigingen-criterion.sparql (hunk 1 above)
[the remark this answers is not in the document]
Comment (wolfderechter) [reply config/migrations/2024/20240715104628-add-verenigingen-criterion.sparql:18]: I see, that makes sense!

**Pull request merged**